lib/Dialect/TritonGPU/Transforms/OptimizeEpilogue.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2023 NVIDIA Corporation & Affiliates. All rights reserved.
```

Comment:
@jayfurmanek I remembered you mentioned something about the copyright? Do we use the same as NV?

Reply:
This file is added in IFU, so I think we do not do anything with it.
Or we should rethink our approach and add AMD copyrights into other files we changed.

P.s. By the way, I just realized that there are no copyright in MFMA specific files...